packages/calcite-components/src/components/combobox-item/combobox-item.tsx

```diff
@@ -85,12 +85,13 @@ export class ComboboxItem implements ConditionalSlotComponent, InteractiveCompon
    * Specifies the selection mode:
    * - `multiple` allows any number of selected items (default),
    * - `single` allows only one selection,
+   * - `single-persist` is like single, but does not allow deselecting,
```

Comment:
nitpick:

For consistency, can we add double quotes to each of the values? For instance:

```
- `"multiple"`....
- `"single"`....
- `"single-persist"`...
- etc.
```